### PR TITLE
Fixes forced optimizer settings in external tests

### PR DESCRIPTION
**Pull request by erak**
@chriseth Discovered that the optimizer settings are not properly passed down to sojc-js.

### CODE REVIEW

File: test/externalTests.sh [hunk 1]
```diff
@@ -150,12 +150,13 @@ function force_solc_settings
...
 
   printLog "Forcing solc settings..."
+  echo "-------------------------------------"
   echo "Config file: $config_file"
   echo "Optimizer settings: $settings"
   echo "EVM version: $evmVersion"
-  echo ""
+  echo "-------------------------------------"
 
-  echo "module.exports['compilers']['solc']['settings'] = { optimizer: $settings, evmVersion: \"$evmVersion\" };" >> "$config_file"
```
Comment (chriseth) [inline test/externalTests.sh]: Actually I would add both. It might be a version problem.

Comment (erak) [reply test/externalTests.sh]: Sounds good!

**Comment by erak** [comment]
Updated.

**Comment by stackenbotten** [comment]
There was an error when running `test_check_style` for commit `c31540fb18d753b86bc0f79f0d1df96d7cc695f7`:
```
Error: Trailing whitespace found:
 test/externalTests.sh:159: # Forcing the settings should always work by just overwriting the solc object. Forcing them by using a 

```
Please check that your changes are working as intended.

**Comment by erak** [comment]
Updated again.

**Review by chriseth — APPROVED** [review]

**Pull request merged**